app/validators/answers/option_answer_validator.py

```diff
@@ -77,3 +79,11 @@ def validate_answer_actions(self):
 
             if block_id and block_id not in self.block_ids:
                 self.add_error(self.BLOCK_ID_MISSING, block_id=block_id)
+
+    def validate_default_mismatch(self):
+        if self.answer.get("default") is not None:
+            values = []
+            for option in self.options:
+                values.append(option.get("value"))
+            if self.answer.get("default") not in values:
+                self.add_error(self.DEFAULT_MISMATCH)
```

Comment:
This could be simplified.
```suggestion
        default_value = self.answer.get("default")
        if default_value and default_value not in [
            option["value"] for option in self.options
        ]:
            self.add_error(self.DEFAULT_MISMATCH)
```

Also, as Rhys mentioned, this will need a unit test.

Reply:
I'm not quite sure on the method naming either. We seem to have a mix of naming it after what it does vs what it should validate against. Maybe the method name should be something like `validate_default_exists_in_options`. Thoughts?

Reply:
there was already a variable called `ANSWER_LABEL_VALUE_MISMATCH` and `validate_default_route()` method so I was trying to use a similar name, but your suggestion is much better.